### PR TITLE
File list style

### LINKED ISSUE #210: file list style broken with long file names
screen shot 2016-08-30 at 11 44 03 am

### PR DESCRIPTION
**Pull request by Kriesse**
This styles the file list to work with the new table markup. Closes #210

### CODE REVIEW

File: public/js/migrate-app.js [hunk 1]
```diff
@@ -68885,11 +68885,9 @@ Tree.prototype.render = function (root, entries, onclick) {
 
   function backRow () {
     if (root === '/' || root === '' || root === '.') return
-    return yo`<li class='entry-back' onclick=${backButton}>
-      <a href='javascript:void(0)'>
-        <span class="name">..</span>
-      </a>
-    </li>`
+    return yo`<tr class='entry-back' onclick=${backButton} href='javascript:void(0)'>
```
Comment (karissa) [inline public/js/migrate-app.js]: i think we don't have to worry about migrate-app.js anymore, it was the old app and we're going to toss it

**Comment by karissa** [comment]
this looks great!

somehow the progress bar css disappeared
screen shot 2016-09-06 at 3 16 17 pm

**Comment by Kriesse** [comment]
@karissa Weird, can't reproduce the screenshot – any hints how to get there? 
Here's what it should look like:

image

**Comment by laurengarcia** [comment]
hm, i can also look into this, too. in my dev environment on master currently there are no styles for the file system and there *are* styles for the import queue.

**Pull request merged**